components/navbar/navbar.jsx

```diff
@@ -354,6 +354,10 @@ export default class Navbar extends React.Component {
         );
     }
 
+    hideHeaderOverlay = () => {
+        this.refs.headerOverlay.hide();
```

Comment:
Make sure the ref exists

Reply:
what I mean here is that you should wrap this in an if statement like

```js
hideHeaderOverlay = () => {
   if (this.refs.headerOverlay) {
      this.refs.headerOverlay.hide();
   }
};
```

Reply:
I totally missed this one. Should be good now.